lib/token.h

```diff
@@ -751,11 +751,11 @@ class CPPCHECKLIB Token {
 
     bool isCChar() const {
         return (((mTokType == eString) && isPrefixStringCharLiteral(mStr, '"', emptyString)) ||
-                ((mTokType ==  eChar) && isPrefixStringCharLiteral(mStr, '\'', emptyString) && mStr.length() == 3));
+                ((mTokType ==  eChar) && isPrefixStringCharLiteral(mStr, '\'', emptyString) && mStr.length() <= 4));
```

Comment:
this looks questionable to me.. I am expecting something like:

    if the char starts with `\` then length should be 4. if the char does not start with `\` the expected length is 3.

What does isCChar() return for `'ab'` ? It should return false.